app/services/return-logs/setup/create-return-submission.service.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+/**
+ * Creates a new return submission, superseding a previous one if it exists
+ * @module CreateReturnSubmissionService
+ */
+
+const { generateUUID, timestampForPostgres } = require('../../../lib/general.lib.js')
+const ReturnSubmissionModel = require('../../../models/return-submission.model.js')
+
+/**
+ * Creates a new return submission. The version number will be 1 if no previous submission exists for this return log
+ * ID; otherwise, the version number will be incremented by 1
+ *
+ * If a previous submission does exist then it will be marked as superseded by setting its `current` property to false
+ *
+ * @param {string} returnLogId - ID of the return log (typically something like v1:6:01/23:987654:2024-11-01:2025-10-31)
+ * @param {string} userId - ID of the user creating the return submission (typically an email address)
+ * @param {string} userType - Type of user (eg. 'internal' or 'external')
```

Comment:
I think you can skip this param. In this context, it will always be `internal`, so it can be defined within this service.